decisions/2019/04/11.md

```diff
@@ -0,0 +1,46 @@
+# Front-end Decision Board, 2019-04-11
+
+## #31 Validation of config objects
+
+**Conclusion**: This is inherently a problem that depends on the
+context.
+
+The general rule for parameters is the relative standard:
+
+- Less than 3 params can be positional
+- 3 and above warrants a configuation object.
```

Comment:
Less than 4 params can be positional
4 and above warrants a configuration object.

Reply:
I've updated to **max 3 positional arguments**.